packages/contracts/test/manager/DiamondTestSetup.sol

```diff
@@ -169,7 +169,7 @@ abstract contract DiamondSetup is DiamondTestHelper {
             "ManagerFacet"
         ];
 
-        // diamod arguments
+        // diamond arguments
```

Comment:
I thought this was fixed in another PR so this is unexpected that I had to manually change again.